apprise/plugins/nextcloud.py

```diff
@@ -297,6 +316,149 @@ def send(self, body, title="", notify_type=NotifyType.INFO, **kwargs):
 
         return not has_error
 
+    def _resolve_targets(self, targets):
```

Comment:
There is a lot going on in here. I need more time to digest this. 

I feel there needs to be a means of identifying a user from a group. Such as groups must be prefixed with a hashtag; e.g
 - `nextcloud://credentials/user1/#group`

There are lots of examples of this in other plugins you can copy from. It also requires you to update the `url()` to correctly reassemble the URL as it was. You need to update the `template_tokem` dictionary to address a new target type too